app/uk/gov/hmrc/servicedependencies/service/SlugDependenciesService.scala

```diff
@@ -86,4 +89,56 @@ class SlugDependenciesService @Inject()(
                             dependency.bobbyRuleViolations.nonEmpty
                           )
     } yield filtered
+
+  private def curatedLibrariesOfSlugInfoFromGraph(slugInfo: SlugInfo, latestVersions: Seq[MongoLatestVersion]) : Future[List[Dependency]] = {
+    for {
+      bobbyRules <- serviceConfigsConnector.getBobbyRules
+      compile    =  curatedLibrariesOfSlugInfoFromGraph(slugInfo, latestVersions, bobbyRules, DependencyScope.Compile)
+      test       =  curatedLibrariesOfSlugInfoFromGraph(slugInfo, latestVersions, bobbyRules, DependencyScope.Test).filterNot(n => compile.exists(_.name == n.name))
+      build      =  curatedLibrariesOfSlugInfoFromGraph(slugInfo, latestVersions, bobbyRules, DependencyScope.Build)
+    } yield compile ++ test ++ build
+  }
+
+  private def curatedLibrariesOfSlugInfoFromGraph(slugInfo: SlugInfo, latestVersions: Seq[MongoLatestVersion], bobbyRules: BobbyRules, scope: DependencyScope = DependencyScope.Compile): List[Dependency] = {
+    val graph = graphParser.parse(dotFileForScope(slugInfo, scope))
+    graph
+      .dependencies
+      .filterNot(_.artefact == slugInfo.name)
+      .flatMap { graphDependency =>
+
+        val latestVersion = latestVersions
+            .find(v => v.group == graphDependency.group && v.name == graphDependency.artefact)
+            .map(_.version)
+
+        Version.parse(graphDependency.version).map { currentVersion =>
+          Dependency(
+              name           = graphDependency.artefact // graphDependency.name is the full unparsed string, not the artifact's name
```

Comment:
does `.name` refer to the field that was renamed `.value`? Might not need the comment then?